src/plugins/dash/videojs-dash.js

```diff
@@ -96,94 +96,15 @@ class Html5DashJS extends Component {
-        // Codec(s) not supported
-        (event.event.id === 'codec') ||
-        // No streams available to stream
-        (event.event.id === 'nostreams') ||
-        // Error creating Stream object
-        (event.event.id === 'nostreamscomposed') ||
-        // syntax error parsing the manifest
-        (event.event.id === 'parse') ||
-        // a stream has multiplexed audio+video
-        (event.event.id === 'multiplexedrep')
-      )) {
-        // These errors have useful error messages, so we forward it on
-        this.player.error({ code: 4, message: event.event.message });
-
-      } else if (event.error === 'mediasource') {
-        // This error happens when dash.js fails to allocate a SourceBuffer
-        // OR the underlying video element throws a `MediaError`.
-        // If it's a buffer allocation fail, the message states which buffer
-        // (audio/video/text) failed allocation.
-        // If it's a `MediaError`, dash.js inspects the error object for
-        // additional information to append to the error type.
-        if (event.event.match('MEDIA_ERR_ABORTED')) {
-          this.player.error({ code: 1, message: event.event });
-        } else if (event.event.match('MEDIA_ERR_NETWORK')) {
-          this.player.error({ code: 2, message: event.event });
-        } else if (event.event.match('MEDIA_ERR_DECODE')) {
-          this.player.error({ code: 3, message: event.event });
-        } else if (event.event.match('MEDIA_ERR_SRC_NOT_SUPPORTED')) {
-          this.player.error({ code: 4, message: event.event });
-        } else if (event.event.match('MEDIA_ERR_ENCRYPTED')) {
-          this.player.error({ code: 5, message: event.event });
-        } else if (event.event.match('UNKNOWN')) {
-          // We shouldn't ever end up here, since this would mean a
-          // `MediaError` thrown by the video element that doesn't comply
-          // with the W3C spec. But, since we should handle the error,
-          // throwing a MEDIA_ERR_SRC_NOT_SUPPORTED is probably the
-          // most reasonable thing to do.
-          this.player.error({ code: 4, message: event.event });
+        if (event.error.code >= 10 && event.error.code <= 35) {
+          this.player.error({ code: 4, dashjsErrorCode: event.error.code, message: event.error.message });
+        } else if (event.error.code < 10) {
+          this.player.error({ code: event.error.code, dashjsErrorCode: event.error.code, message: event.error.message });
+        } else if (event.error.code >= 100 && event.error.code <= 114) {
+          this.player.error({ code: 5, dashjsErrorCode: event.error.code, message: event.error.message });
         } else {
-          // Buffer allocation error
-          this.player.error({ code: 4, message: event.event });
+          this.player.error({ code: event.error.code, message: event.error.message });
         }
-
-      } else if (event.error === 'capability' && event.event === 'encryptedmedia') {
```

Comment:
why do we remove all this? whats the effect?

Reply:
no effect this from the old implmentation